src/components/Modals/Onboarding/Onboarding.vue

```diff
@@ -185,11 +188,11 @@ export default defineComponent({
       if (this.name.lastName === '') {
         messages.push('a last name');
       }
-      if (this.onboarding.gradYear === '') {
-        messages.push('a graduation year');
+      if (this.onboarding.gradYear === '' || !this.onboarding.gradSem) {
+        messages.push('a graduation date');
       }
-      if (this.onboarding.entranceYear === '') {
-        messages.push('an entrance year');
+      if (this.onboarding.entranceYear === '' || !this.onboarding.entranceSem) {
+        messages.push('an entrance date');
```

Comment:
small: can we flip the order of this so the order of items in the error message matches the order of items in the modal?
i.e. display "Please select an entrance date and a graduation date."